singlestoredb_cloud/manifest.json

```diff
@@ -0,0 +1,55 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "c7638089-0864-4ddc-bd32-b731c58fe567",
+  "app_id": "singlestoredb-cloud",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Send your SinglestoreDB Cloud metrics to DataDog.",
+    "title": "SingleStoreDB Cloud",
+    "media": [
+      {
+        "media_type": "image",
+        "caption": "SinglestoreDB Cloud - Dashboard",
+        "image_url": "images/singlestoredb-cloud-dashboard.png"
```

Comment:
This image looks a bit stretched - would you be able to instead paste the original screenshot on a background that has a 16:9 aspect ratio? You can find our [media guidelines here](https://docs.datadoghq.com/developers/integrations/create_a_tile/?tab=buildatileontheintegrationspage#media-carousel)

Reply:
Updated the image by pasting the original screenshot on a background that has 16:9 aspect ratio